maven-compat/src/main/java/org/apache/maven/profiles/activation/OperatingSystemProfileActivator.java

```diff
@@ -122,12 +153,54 @@ private boolean determineFamilyMatch(String family) {
             test = test.substring(1);
         }
 
-        boolean result = Os.isFamily(test);
+        boolean result = isFamily(test);
 
         if (reverse) {
             return !result;
         } else {
             return result;
         }
     }
+
+    private boolean isFamily(String family) {
```

Comment:
This can likely be made static, though I wonder if there's more work to be done here. Netware? Windows 98? It amazes me that in one thread I'm trying to convince folks not to drop support for the widely used Java 8 while over here in this PR I'm seeing support for OSs that were defunct circa Java 1.4.

Reply:
Mmh, could the reason be that you're considering this code as new code, whereas I consider it as a code move from plexus-utils ?  If it was good to be used there, I assume it's good to be used in maven too as-is... 

So which ones do you want to remove support for exactly ?

Reply:
Just moving code from plexus is not kosher, either technically or legally. plexus-utils has iffy ownership and license status due to the Codehaus mess. Nor has it had good maintenance or updates over the years. It should be replaced de novo. Copy paste of the existing code is not OK.

Reply:
> Just moving code from plexus is not kosher, either technically or legally. plexus-utils has iffy ownership and license status due to the Codehaus mess. Nor has it had good maintenance or updates over the years. It should be replaced de novo. Copy paste of the existing code into this project is not OK.

@CrazyHZM for `Os`, we need to use [the one from maven-shared-utils instead](https://github.com/apache/maven-shared-utils/blob/master/src/main/java/org/apache/maven/shared/utils/Os.java)

Reply:
> > Just moving code from plexus is not kosher, either technically or legally. plexus-utils has iffy ownership and license status due to the Codehaus mess. Nor has it had good maintenance or updates over the years. It should be replaced de novo. Copy paste of the existing code into this project is not OK.
> 
> @CrazyHZM for `Os`, we need to use [the one from maven-shared-utils instead](https://github.com/apache/maven-shared-utils/blob/master/src/main/java/org/apache/maven/shared/utils/Os.java)

@gnodet Done.